include/ccf/receipt.h

```diff
@@ -51,6 +54,8 @@ namespace ccf
     std::optional<std::string> leaf = std::nullopt;
     /// Leaf components in transactions emitted by 2.x networks.
     std::optional<LeafComponents> leaf_components = std::nullopt;
+
```

Comment:
```suggestion
/// Service certificate (PEM) endorsing the node certificate (cert). Unset if the service never experienced disaster recovery.
```

Or something like that?